tox.ini

```diff
@@ -1,26 +1,25 @@
 [tox]
-envlist = py26, py27, py33, py34, py35, flake8
-
-[travis]
-python =
-    3.5: py35
-    3.4: py34
-    3.3: py33
-    2.7: py27
-    2.6: py26
-
-[testenv:flake8]
-basepython=python
-deps=flake8
-commands=flake8 pystac
+envlist = py34, py35, py36
 
 [testenv]
-setenv =
-    PYTHONPATH = {toxinidir}
 
-commands = python setup.py test
+commands =
+    check-manifest --ignore tox.ini
+    {envpython} setup.py check -m -r -s
+    #    flake8 .
```

Comment:
I think that we should edit this command (even if left commented-out) to:

```
flake8 --exclude ".svn,CVS,.bzr,.hg,.git,__pycache__,.tox,.eggs,*.egg,docs"
```

This helps remove a bunch of stuff that has little to do with the project source code.

Reply:
As far as improving this, I would add `__init__.py` and `build` as well (at least helps reduce issues for local development):

`flake8 --exclude "__init__.py,build,.svn,CVS,.bzr,.hg,.git,__pycache__,.tox,.eggs,*.egg,docs"`